core/mondoo-kubernetes-best-practices.mql.yaml

```diff
@@ -1977,6 +1981,29 @@ queries:
         _['ports'] == null || _['ports'].all(_['hostPort'] == null)
       }
     }
+- uid: mondoo-kubernetes-best-practices-ingress-cert-expiration
+  title: Ingress certificates should be renewed
```

Comment:
```suggestion
  title: Ingress certificates nearing expiration
```

Reply:
I'm happy to update the text here, and while this query will/should fire when the cert expiration is less than 15 days away, it will also continue to fire after the cert has already expired.

So the initial firing/alert would be as it is "nearing expiration", it can/will fire/alert after full expiration.

Or should we have a query with a smaller impact for `expiration > 15days` and a more severe impact query for `expiration < 0days` (ie. already expired)?

WDYT?

Reply:
What about this updated title?